plugins/woocommerce/client/blocks/assets/js/blocks/add-to-cart-with-options/quantity-selector/edit.tsx

```diff
@@ -1,62 +1,16 @@
 /**
  * External dependencies
  */
-import { InspectorControls, useBlockProps } from '@wordpress/block-editor';
+import { useBlockProps } from '@wordpress/block-editor';
 import { __ } from '@wordpress/i18n';
-import { BlockEditProps } from '@wordpress/blocks';
 import { useSelect } from '@wordpress/data';
 import { isSiteEditorPage } from '@woocommerce/utils';
-import {
-	Disabled,
-	PanelBody,
-	// eslint-disable-next-line @typescript-eslint/ban-ts-comment
-	// @ts-ignore - Ignoring because `__experimentalToggleGroupControl` is not yet in the type definitions.
-	// eslint-disable-next-line @wordpress/no-unsafe-wp-apis
-	__experimentalToggleGroupControl as ToggleGroupControl,
-	// eslint-disable-next-line @typescript-eslint/ban-ts-comment
-	// @ts-ignore - Ignoring because `__experimentalToggleGroupControl` is not yet in the type definitions.
-	// eslint-disable-next-line @wordpress/no-unsafe-wp-apis
-	__experimentalToggleGroupControlOption as ToggleGroupControlOption,
-} from '@wordpress/components';
-
-enum QuantitySelectorStyle {
-	Input = 'input',
-	Stepper = 'stepper',
-}
-
-interface Attributes {
-	className?: string;
-	quantitySelectorStyle: QuantitySelectorStyle;
-}
-
-const getHelpText = ( quantitySelectorStyle: QuantitySelectorStyle ) => {
-	if ( quantitySelectorStyle === QuantitySelectorStyle.Input ) {
-		return __(
-			'Shoppers can enter a number of items to add to cart.',
-			'woocommerce'
-		);
-	}
-	if ( quantitySelectorStyle === QuantitySelectorStyle.Stepper ) {
-		return __(
-			'Shoppers can use buttons to change the number of items to add to cart.',
-			'woocommerce'
-		);
-	}
-};
-
-const AddToCartWithOptionsQuantitySelectorEdit = (
-	props: BlockEditProps< Attributes >
-) => {
-	const { setAttributes } = props;
-	const { quantitySelectorStyle } = props.attributes;
-
-	const quantitySelectorStyleClass =
-		quantitySelectorStyle === QuantitySelectorStyle.Input
-			? 'wc-block-add-to-cart-with-options__quantity-selector--input'
-			: 'wc-block-add-to-cart-with-options__quantity-selector--stepper';
+import { Disabled } from '@wordpress/components';
 
+const AddToCartWithOptionsQuantitySelectorEdit = () => {
 	const blockProps = useBlockProps( {
-		className: `wc-block-add-to-cart-with-options__quantity-selector ${ quantitySelectorStyleClass }`,
+		className:
+			'wc-block-add-to-cart-with-options__quantity-selector wc-block-add-to-cart-with-options__quantity-selector--stepper',
```

Comment:
Same here, given that we aren't using the `wc-block-add-to-cart-with-options__quantity-selector--stepper` for styling, let's remove it. `wc-block-add-to-cart-with-options__quantity-selector` should be enough.